internal/restapi/server/server.go

```diff
@@ -30,14 +34,45 @@ func (srv *Server) Service() *web.Service {
 	bearerAuth := &BearerTokenAuth{AuthToken: srv.AuthToken}
 	s.Wrap(bearerAuth.Middleware)
 
-	s.Post("/api/v0/scan", usecase.NewInteractor(scan.Post))
-	s.Get("/api/v0/scan/{scan_id}", usecase.NewInteractor(scan.Get))
+	// Define API endpoints
+	s.Get("/api/v0/integrations", usecase.NewInteractor(integrations.ListIntegrations(srv.DB)))
+	s.Get("/api/v0/integration/{id}", usecase.NewInteractor(integrations.GetIntegration(srv.DB)))
+	s.Post("/api/v0/integration/create", usecase.NewInteractor(integrations.CreateIntegration(srv.DB)))
+	s.Put("/api/v0/integration/update/{id}", usecase.NewInteractor(integrations.UpdateIntegration(srv.DB)))
```

Comment:
Using plural nouns is more inline with the REST conventions: https://www.perplexity.ai/search/rest-api-should-we-use-singula-fhKg8XFxQPKj19Orddblow

Also, it should be okay to drop the `/create` and `/update` words in the route, because we denote that via the HTTP methods in our REST API.

```suggestion
s.Get("/api/v0/integrations/{id}", usecase.NewInteractor(integrations.GetIntegration(srv.DB)))
s.Post("/api/v0/integrations", usecase.NewInteractor(integrations.CreateIntegration(srv.DB)))
s.Put("/api/v0/integrations/{id}", usecase.NewInteractor(integrations.UpdateIntegration(srv.DB)))
```